<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShotsWindow</class>
 <widget class="QDialog" name="ShotsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="penaltyEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="competitionButton">
     <property name="text">
      <string>Võistlus</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTableView" name="shotsTable"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="penaltyButton">
     <property name="text">
      <string>Sisesta karistus</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="shotsLabel">
     <property name="text">
      <string>Kõik saabunud lasud:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="ignoreButton">
     <property name="text">
      <string>Ignoreeri</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="insertShotButton">
     <property name="text">
      <string>Sisesta lask</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="sightersButton">
     <property name="text">
      <string>Proov</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="karistusSilt">
     <property name="text">
      <string>Karistus:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Sulge</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QToolButton" name="downButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="upButton">
     <property name="text">
      <string>^</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="deleteButton">
     <property name="text">
      <string>Kustuta</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ShotsWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>598</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
